src/main/kotlin/org/malachite/estella/process/domain/exceptions.kt

```diff
@@ -1,5 +1,8 @@
 package org.malachite.estella.process.domain
 
```

Comment:
Wgl ja bym tutaj wprowadził jakiś ```class RecruitmentProcessException(override val message: String): Exception()```, po którym dziedziczyłyby te exceptiony. I tak wszystkie handlery to `ResponseEntity.badRequest().body(Message(???))`, więc czemu nie pobierać zawsze `message`, który już będzie uformowany i zrobić z tego jeden handler